### ISSUE TITLE
enquiry of nature of smaller data set

### ISSUE THREAD

**Issue by lolbus** [hugochan/Eye-Tracker#22]
Hi Hugo Chan,

I'm a research staff based in Singapore, Nanyang Tech University.

I came across your version of iTracker am I'm really interested to study it and studying if I can implement it in the computer games I'm working on.

In the original paper of Eye Tracker for Everyone, it was mentioned that the number of subjects in the training set matters a lot to achieve generalization of the neural network. May I check with you while processing a smaller subset of the original data, how many subjects were included (Out of the original 1437 subjects) how many instances per subject?

Does the validation set share similar subjects and instances per subjects as the training set or does the validation set only include subjects not seen by the model when being trained by the training set?

Possible to share your code how you processed the original data set? :)

That aside, 
Currently, I made an inference engine using openCV library but unfortunately using your pre-trained model as the deployed model, and my face through my laptop's webcam as the Inference instance, accuracy is horrible. The pre-trained model 23 was not able to detect my gaze at all. It gave random values.

What do you think may be the problem?
 
Guang Way
